pyLibrary/queries/expressions.py

```diff
@@ -2633,9 +2633,11 @@ def to_python(self, not_null=False, boolean=False, many=False):
     def to_sql(self, schema, not_null=False, boolean=False):
         if not isinstance(self.superset, Literal):
             Log.error("Not supported")
-        var = self.value.to_sql(schema)
-        return " OR ".join("(" + var + "==" + sql_quote(v) + ")" for v in json2value(self.superset))
-
+        var = self.value.to_sql(schema,boolean=boolean)
+        if json2value(self.superset):
```

Comment:
maybe define a new variable `superset = json2value(self.superset.json)`. I added `.json` to be explicit about what is being converted.